core/templates/dev/head/profile/profile.html

```diff
@@ -49,13 +49,224 @@ <h1 class="oppia-profile-username"><strong>{{PROFILE_USERNAME}}</strong></h1>
           </div>
         </div>
       </div>
+        <div ng-cloak="" class="oppia-profile-tabs" ng-app="MyApp">
```

Comment:
deindent this line by 2; it's a sibling of the previous div. Remove ng-app; you don't need it here. (It's only needed for the top-level oppia app.)
Then, in the following lines: please ensure indentation is 2 spaces, not 4. Otherwise it's rather spread out and quite hard to read. I'll do a further review once you've fixed this.